CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ## 22.4.0-RC2
 
 ### Additions and Improvements
+- Onchain node permissioning - log the enode that was being checked with IllegalStateException. [#3697](https://github.com/hyperledger/besu/pull/3697)
```

Comment:
Could this be more descriptive? Something like:
"log the enodeURL that was previously only throwing an IllegalStateException during the isPermitted check"

Reply:
done